testdata/fixtures/api.omise.co/receipts-get.json

```diff
@@ -0,0 +1,36 @@
+{
+    "object": "list",
+    "from": "2017-07-01T00:00:00Z",
+    "to": "2017-07-31T23:59:59Z",
+    "offset": 0,
+    "limit": 20,
+    "total": 3,
```

Comment:
Not sure if this is intended but the `total` number is 3 but you have only 1 receipt object in the `data` list